source/reference/minio-server/settings/console.rst

```diff
@@ -176,6 +178,148 @@ Log Query URL
       This setting does not have a configuration variable setting.
       Use the Environment Variable instead.
 
+Content Security Policy
+~~~~~~~~~~~~~~~~~~~~~~~
+
+*Optional*
+
+Set the `Content-Security-Policy <https://en.wikipedia.org/wiki/Content_Security_Policy>`__ header in MinIO Console responses.
+Defaults to ``default-src 'self' 'unsafe-eval' 'unsafe-inline';``
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_BROWSER_CONTENT_SECURITY_POLICY
+
+      .. code-block:: shell
+         :class: copyable
+
+         set MINIO_BROWSER_CONTENT_SECURITY_POLICY="default-src 'self' 'unsafe-eval' 'unsafe-inline';"
+
+   .. tab-item:: Configuration Setting
+      :sync: config
+
+      .. mc-conf:: browser csp_policy
+         :delimiter: " "
+
+      .. code-block:: shell
+
+         mc admin config set browser \
+            csp_policy="default-src 'self' 'unsafe-eval' 'unsafe-inline';" \
+            [ARGUMENT=VALUE ...]
+
+
+Strict Transport Security
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Optional*
+
+Set the `Strict-Transport-Security <https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security>`__ header in MinIO Console responses.
+Set the appropriate environment variable or configuration settings according to the HSTS requirements for your environment.
```

Comment:
Is there a more useful description? Do all these need to be set in order for the Strict-Transport-Security header to be created? The feature PR notes there is no default value.

Reply:
Deferring to @pjuarezd here, but maybe @kaankabalak or @rushenn might have some insight as well

Reply:
key setting is `hsts_seconds` if seconds is "0" then the header is not rendered regardless of values on other settings 

example 1 `mc admin config set play browser hsts_seconds=500` will render

```
Strict-Transport-Security: max-age=500
```

example 2 `mc admin config set play browser hsts_seconds=500 hsts_include_subdomains=on` will render

```
Strict-Transport-Security: max-age=500; includeSubDomains
```

example 3 `mc admin config set play browser hsts_seconds=500 hsts_include_subdomains=on hsts_preload=on` will render

```
Strict-Transport-Security: max-age=500; includeSubDomains; preload
```

Reply:
These examples help a ton @pjuarezd 🎉